### PR TITLE
Adding a new project for beginners

### PR DESCRIPTION
**Pull request by scamacho23**
A new project that could really use some help. Very open to new programmers/developers/github users. I think that this would a great addition to the up-for-grabs page. It's a low stress project so any help is very welcome and appreciated

### CODE REVIEW

File: _data/projects/quicknote.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: Quick Note
+desc: Quick Note is an open-source project for saving snippets of information to 'memory' files from the command line
+site: https://github.com/scamacho59/quicknote
```
Comment (shiftkey) [inline _data/projects/quicknote.yml]: :art: point to renamed repository?

```suggestion
site: https://github.com/scamacho23/quicknote
```

**Review by shiftkey — COMMENTED** [review]
Thanks for the contribution @scamacho23! 

Just one suggestion, otherwise this is ready to merge...

**Comment by scamacho23** [comment]
Thanks for the suggestion, @shiftkey!

File: _data/projects/quicknote.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name: Quick Note
+desc: Quick Note is an open-source project for saving snippets of information to 'memory' files from the command line
+site: https://github.com/scamacho23/quicknote
```
Comment (shiftkey) [inline _data/projects/quicknote.yml]:
```suggestion
site: https://github.com/scamacho23/homebrew-quicknote
```

File: _data/projects/quicknote.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+tags:
+  - python
+  - shell
+upforgrabs:
+  name: help wanted
+  link: https://github.com/scamacho23/quicknote/labels/help%20wanted
```
Comment (shiftkey) [inline _data/projects/quicknote.yml]:
```suggestion
  link: https://github.com/scamacho23/homebrew-quicknote/labels/help%20wanted
```

**Review by shiftkey — COMMENTED** [review]
Looks like it's been renamed since the last review:

**Comment by scamacho23** [comment]
Thanks for making those changes, @shiftkey

**Review by shiftkey — APPROVED** [review]

**Pull request merged**